src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.scss

```diff
@@ -0,0 +1,5 @@
+@use './src/styles/vendor/variables' as vv;
+
+.custom-radio {
+    padding-left: 35px;
+}
```

Comment:
Can we instead use classes like `pl-2` or `pl-3` right next to `custom-radio` or a `div` above it?